controllers/defaults/resources.go

```diff
@@ -146,6 +166,26 @@ var (
 				corev1.ResourceMemory: resource.MustParse("2Gi"),
 			},
 		},
+		"crashcollector": {
```

Comment:
Let's keep the resources the same across all the profiles, no need to change them since it's such a simple background daemon that doesn't impact performance of the cluster.